lib/gems/pending/appliance_console.rb

```diff
@@ -307,7 +307,7 @@ module ApplianceConsole
             system_hosts.hostname = new_host
             system_hosts.set_loopback_hostname(new_host)
             system_hosts.save
-            LinuxAdmin::Service.new("network").restart
+            LinuxAdmin::Service.new("NetworkManager").restart
```

Comment:
The hostnamectl command which we're running seems to take care of everything and not require a reboot.
https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Networking_Guide/sec_Configuring_Host_Names_Using_hostnamectl.html